static.json

```diff
@@ -3,5 +3,13 @@
     "clean_urls": false,
     "routes": {
       "/**": "index.html"
+    },
+    "proxies": {
+      "/api/tasks": {
+        "origin": "https://todo-mongo-graphql-server.herokuapp.com"
+      },
+      "/api/grackle-demo": {
+        "origin": "https://${GRACKLE_DEMO_BASE}"
```

Comment:
Does this come from heroku?

Reply:
We define it as an environment variable in Heroku. This allows us to use the same app pointing to different servers in staging and production.